### PR TITLE
Support copy from right in timetable editor

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Timetable editor has a cool keyboard shortcut to copy from left/top, but nothing to copy from right! We add shift+l to do this, mirroring shift+'.

Please test thoroughly I could only come up with so many edge cases. Tempted to add a new e2e case

### CODE REVIEW

**Review by josh-willis-arcadis — APPROVED** [review]

File: lib/editor/util/timetable.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { EXCEPTION_EXEMPLARS } from '.'
 export const SHORTCUTS = {
   offset: ['o', 'SHIFT:+:o', 'i', 'SHIFT:+:i', '-', 'SHIFT:+:-', '+', 'SHIFT:+:+'],
   navigate: ['k:/:j', '←:/:→', 'x', 'a', 'd'],
-  modify: ['#', 'n', 'c', 'SHIFT:+:\'', 'SHIFT:+:;']
+  modify: ['#', 'n', 'c', 'SHIFT:+:\'', 'SHIFT:+:l', 'SHIFT:+:;']
```
Comment (binh-dam-ibigroup) [inline lib/editor/util/timetable.js:15]: ~Is that me, or was copy-from-left SHIFT+Apostrophy? (SHIFT+semicolon does nothing.)~

File: lib/editor/components/timetable/TimetableGrid.js [hunk 2]
```diff
@@ -205,6 +205,17 @@ export default class TimetableGrid extends Component<Props> {
         }
         break
       }
+      case 76: { // l
+        // Set current cell's value to adjacent cell's (rightward) value.
+        const adjacentCol = columns[this._getColIndex(scrollToColumn) + 1]
+        if (evt.shiftKey && !activeCell && isTimeFormat(adjacentCol.type)) {
```
Comment (binh-dam-ibigroup) [inline lib/editor/components/timetable/TimetableGrid.js]: Need to check that `adjacentCol` is defined (see console error), and that check needs to be added for copy-from-left.

Comment (binh-dam-ibigroup) [reply lib/editor/components/timetable/TimetableGrid.js]: I think there should also be an `isTimeFormat` check for the focused cell. (We probably don't want to copy a time value into a non-time cell, but this is possible, as written, from the first time cell into the trip short name cell).

Comment (binh-dam-ibigroup) [reply lib/editor/components/timetable/TimetableGrid.js]: There is a caveat that the adjacent cells must be visible for the copy operations to complete, but that is not blocking for this PR.

File: lib/editor/util/timetable.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/editor/util/timetable.js:15]: > Is that me, or was copy-from-left SHIFT+Apostrophy? (SHIFT+semicolon does nothing.)

Disregard^^

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Add null checks and a check to avoid copying time into a non-time cell.

File: lib/editor/components/timetable/TimetableGrid.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/editor/components/timetable/TimetableGrid.js]: Good ideas! Really this entire file should be refactored but these are great suggestions for now1

**Review by binh-dam-ibigroup — APPROVED** [review]
Thanks for the changes
